data/assets/examples/renderable/renderabledisc/disc_hollow.asset

```diff
@@ -0,0 +1,30 @@
+-- Hollow
```

Comment:
How about "ring" instead of "hollow"? "hollow" to me sounds like a 3d object which is not filled ("ihålig"). Or "Disc with hole"

Reply:
I would prefer "Disc With Hole" in that case 🤔

Reply:
Although ,technically, the hollow disc _is actually_ a 3D object which is not filled ;) So personally I find that name pretty suitable

Google seems to do so too. But I'm fine with changing it 
![image](https://github.com/user-attachments/assets/ebc5c325-0857-4a34-b80e-555ee09b7fad)

Reply:
I'd go with "hollow disk" "disk with hole" as well.  "Ring" sound more 3Dish to me than "Disk"

Reply:
CHanged to "With Hole"